guides/common/modules/ref_prerequisites-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,5 @@
+[id="prerequisites-disaster-recovery-with-two-active-project-servers"]
+= Prerequisites
+
+* Review xref:overview-of-recommended-disaster-recovery-plans[] to make sure that this disaster recovery plan works for you.
+* You have a {ProjectServer} installed.
```

Comment:
Don't you need _two_ to follow this? :shrug:

Reply:
The second one is created as part of the first procedure (`Preparing...`). My reasoning was that I want to write this for users who currently have a single server and are thinking of how to best prepare for DR. That's where they are starting their DR journey so I wanted to make the process of getting their second server a part of the procedure rather than a prerequisite.